### PR TITLE
make new lifesaver less CBT (now with pill bottles)

### PR DESCRIPTION
**Pull request by fernandos13**
## About The Pull Request
adds a "new" version of the lifesaver bag
**exactly the same** item but this one **can only be retrieved from the medic corpsman vendor** one single time per medic
but starts with the following differences from the regular lifesaver bag:
+ (-) quick clot injector from lifesaver belt
+ (-) 3 combat injectors from lifesaver belt
+ (+) quick clot pillbottle to the lifesaver belt
+ (+) 1 brute kit, to lifesaver belt.
+ (+) 1 burn kit, to lifesaver belt.
+ (+) 1 splint, to lifesaver belt.

_(-) item removed
(+) item added_

**also** adds the _ADV_ medkit to the essential kit of medic "with the beacon. oxy hypo, and rollerbed, ETC.".

## Why It's Good For The Game
mostly for a QoL change. we all spend time in the setting up of our classes. but when you see the corpsman room FILLED with combat autoinjectors litter. and corpmans rushing to get pillbottles from other vendors. ALWAYS. why not give em that already. and leave the injectors as the secondary option.
plus essential kit. should be the "ESSENTIAL" medic kit. so why not add a little ADV medkit to it. its not something overly expensive but could help medics to get a reserve of the 3 basic chems. and healing items. 

## Changelog
:cl:
add: new lifesaver medic belt in the corpsman belt selection
tweak:the medics ESSENTIAL kit, to have one advanced medkit.
/:cl:

### CODE REVIEW

**Comment by Fikou** [comment]
i dont think that changelog will work

**Comment by Chaznoodles** [comment]
Bruh that's going to make the changelog commit suicide

**Comment by fernandos13** [comment]
how do you guys want the changelog.. ?
i did with 1 single line as 
tweak: 
and people disliked 
so i made the detailed way
and.. people disliked

its ok.. but how do you guys want me to do it?

**Comment by jroinc** [comment]
It's INTENTIONAL that the CLB has injectors instead of pills.

If you want to customize it ingame, be my guest, but it's balanced that way intentionally.

**Comment by fernandos13** [comment]
i mean.. i see EVERY medic i come across dropping ALL the injectors... whats the balance in it? (not saying theres no balance i just don't see it)
feels.. inconvenient. to have to pick two belts EVERY round. just to steal pillbottle of the exact chems you have. but in injector form

**Comment by jroinc** [comment]
The balance isn't for medics, it's for all the belts that start roundstart on the station.

An UNMODDED CLB should be best at quick-heals, and an UNMODDED medbelt should be best at stabilizing and fixing problems, but medics can obviously customize the loadout as they see fit. PFC Bobby who just grabs one cause he wants to be medic shouldn't get that choice easily, without being a medic or a hacker in medbay.

That said, I wouldn't be opposed to you making this a child of the CLB with your loadout, and making the medic vendors (and only the medic vendors) give the upgraded one.

**Comment by fernandos13** [comment]
oh i see the problems is that i oversighted the PFCs, well then i'll make the child and give it to medics. nice one thanks Jronic. Would change the regular combat injectors to adv injectors too much "sounds ok IN MY HEAD since only medics would be able to get roundstart" but is it

File: code/game/objects/machinery/vending/new_marine_vendors.dm [hunk 1]
```diff
@@ -451,7 +451,7 @@ GLOBAL_LIST_INIT(marine_selector_cats, list(
 		/obj/item/clothing/tie/storage/webbing = list(CAT_WEB, "Tactical webbing", 0, "black"),
 		/obj/item/clothing/tie/holster = list(CAT_WEB, "Shoulder handgun holster", 0, "black"),
 		/obj/item/clothing/tie/storage/white_vest = list(CAT_WEB, "Corpsman surgical vest", 0, "black"),
-		/obj/item/storage/belt/combatLifesaver = list(CAT_BEL, "Lifesaver belt", 0, "orange"),
+		/obj/item/storage/belt/medicLifesaver = list(CAT_BEL, "Lifesaver medical belt", 0, "orange"),
 		/obj/item/storage/belt/medical = list(CAT_BEL, "Medical belt", 0, "black"),
```
Comment (TiviPlus) [inline code/game/objects/machinery/vending/new_marine_vendors.dm:455]:
```suggestion
		/obj/item/storage/belt/medicLifesaver = list(CAT_BEL, "Lifesaver medic belt", 0, "orange"),
		/obj/item/storage/belt/medical = list(CAT_BEL, "Medical belt", 0, "black"),
```
for clarity

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**